content/arkly.ttl

```diff
@@ -3,5 +3,6 @@
 :arkly
     skos:prefLabel "arkly"@en;
     skos:definition '''
-A decentralized digital archive and permanent storage solution.
+      A next-generation decentralized digital archives
+      management platform and permanent storage solution.
     ''' .
```

Comment:
Do we want to add an external ref?

Reply:
why not? We are pulling the language largely from the about page